charts/csm-authorization/templates/ingress.yaml

```diff
@@ -48,4 +48,13 @@ spec:
               port:
                 number: 8080
   {{- end }}
-  {{- end}}
+  {{- end }}
+  - http:
```

Comment:
Now that we can use the IP address of the cluster master node, should this be mentioned as a comment in the values file for `authorization.hostname` value?

Reply:
Updated.